Changelog.md

```diff
@@ -86,6 +87,7 @@
         * [Pull request #317](https://github.com/ignitionrobotics/ign-math/pull/317)
     1. Quaternion
         * [Pull request #324](https://github.com/ignitionrobotics/ign-math/pull/324)
+        * [Pull request #361](https://github.com/ignitionrobotics/ign-math/pull/361)
```

Comment:
this is in two places?

Reply:
Yeah because that PR affected both Quaternion and PID